### PR TITLE
Run CAPI e2e on AWS

### PR DESCRIPTION
**Pull request by alexander-demicev**
This PR includes two commits:
1. Improvement of a script that waits for CAPI to be deployed. Now it will wait for all required pods.
2. Run CAPI e2e suite after core one.

### CODE REVIEW

File: ci-operator/step-registry/capi/conf/apply-feature-gate/capi-conf-apply-feature-gate-commands.sh [hunk 1]
```diff
@@ -72,12 +83,13 @@ function execute() {
       return
     fi
 
+    echo ${CMD}
```
Comment (lobziik) [inline ci-operator/step-registry/capi/conf/apply-feature-gate/capi-conf-apply-feature-gate-commands.sh:86]: Does it needed, or just debug leftover?

File: ci-operator/step-registry/openshift/e2e/aws/capi/openshift-e2e-aws-capi-workflow.yaml [hunk 2]
```diff
@@ -2,15 +2,15 @@ workflow:
   as: openshift-e2e-aws-capi
   steps:
     pre:
-    - ref: ipi-install-hosted-loki
-    - ref: ipi-conf
-    - ref: ipi-conf-aws
-    - chain: ipi-install
-    - ref: capi-conf-apply-feature-gate
+      - ref: ipi-install-hosted-loki
```
Comment (JoelSpeed) [inline ci-operator/step-registry/openshift/e2e/aws/capi/openshift-e2e-aws-capi-workflow.yaml:5]: This file seems to only have whitespace changes, perhaps we can drop the changes from this file?

**Pull request merged**